Mailer/app/api/v1/Handlers/Users/IsUserUnsubscribedHandler.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Users;
+
+use Nette\Utils\Json;
+use Nette\Utils\JsonException;
+use Remp\MailerModule\Repository\ListsRepository;
+use Remp\MailerModule\Repository\LogsRepository;
+use Remp\MailerModule\Repository\MailTypesRepository;
+use Remp\MailerModule\Repository\UserSubscriptionsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Params\InputParam;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tracy\Debugger;
+
+class IsUserUnsubscribedHandler extends BaseHandler
+{
+    private $userSubscriptionsRepository;
+
+    public function __construct(
+        UserSubscriptionsRepository $userSubscriptionsRepository
+    ) {
+        parent::__construct();
+        $this->userSubscriptionsRepository = $userSubscriptionsRepository;
+    }
+
+    public function params()
+    {
+        return [
+            new InputParam(InputParam::TYPE_POST, 'user_id', InputParam::REQUIRED),
```

Comment:
Zatial som to nevalidoval cez json schemu, poddla mna je pri takom malom pocte vstupnych atributov prehladnejsi tento povodny pristup.